src/wp-includes/class-wp-script-modules.php

```diff
@@ -9,61 +9,63 @@
-	 *                                                                                 WordPress root directory.
-	 * @param array<string|array{id: string, import?: 'static'|'dynamic' }> $deps      Optional. An array of module
-	 *                                                                                 identifiers of the dependencies of
-	 *                                                                                 this module. The dependencies can
-	 *                                                                                 be strings or arrays. If they are
-	 *                                                                                 arrays, they need an `id` key with
-	 *                                                                                 the module identifier, and can
-	 *                                                                                 contain an `import` key with either
-	 *                                                                                 `static` or `dynamic`. By default,
-	 *                                                                                 dependencies that don't contain an
-	 *                                                                                 `import` key are considered static.
-	 * @param string|false|null                                             $version   Optional. String specifying the
-	 *                                                                                 module version number. Defaults to
-	 *                                                                                 false. It is added to the URL as a
-	 *                                                                                 query string for cache busting
-	 *                                                                                 purposes. If $version is set to
-	 *                                                                                 false, the version number is the
-	 *                                                                                 currently installed WordPress
-	 *                                                                                 version. If $version is set to
-	 *                                                                                 null, no version is added.
+	 * @param string            $id       The identifier of the script module. Should be unique. It will be used in the
+	 *                                    final import map.
+	 * @param string            $src      Optional. Full URL of the script module, or path of the script module relative
+	 *                                    to the WordPress root directory. If it is provided and the script module has
+	 *                                    not been registered yet, it will be registered.
+	 * @param array<string|array{
+	 *            id: string,
+	 *            import?: 'static'|'dynamic'
+	 *        }>                $deps     {
```

Comment:
I didn't get any love for my [proposal](https://wordpress.slack.com/archives/C5VCTJGH3/p1705533442440489) to include the array shape. So probably best to remove for now:

```suggestion
	 * @param array             $deps     {
```